src/custom-elements/src/components/popover/popover.tsx

```diff
@@ -34,6 +34,11 @@ export class Popover {
    */
   @Prop({ reflect: true }) arrow: boolean;
 
+  /**
+   * to add a close button to the popover
+   */
+  @Prop({ reflect: true }) closable: boolean;
```

Comment:
Is this a mandatory field? Or can it take a default value if not provided?

Reply:
Not a mandatory field.
Should we make it
```suggestion
  @Prop({ reflect: true }) closable = false;
```
?

Reply:
Yes, I think it'll be cleaner for the developer